Gemfile.lock

```diff
@@ -210,7 +210,7 @@ GEM
     turbolinks-source (5.0.3)
     tzinfo (1.2.4)
       thread_safe (~> 0.1)
-    uglifier (3.2.0)
+    uglifier (4.0.1)
```

Comment:
why gemfile.lock changed but gemfile wasn't?
usually they have to be change simultaneously